src/shared/utils.ts

```diff
@@ -78,6 +79,9 @@ export const makeFilteredArraySchema = <T extends ZodSchema>(schema: T) =>
 
             if (!parsedItem.success) {
                 logger.warn("Error parsing item", parsedItem.error.format());
+                putMetricData(`custom/SharedUtils-${process.env.STAGE}`, [
+                    { MetricName: "makeFilteredArraySchemaParseError", Value: 1 },
```

Comment:
Generally I think the custom for metric names is to use Pascal Case for metric names, so `MakeFilteredArraySchemaParseError` in this case